azext_iot/_params.py

```diff
@@ -30,531 +30,1035 @@
-                         help='Indicates whether to expand the device capability model\'s'
-                              ' interface definitions or not.')
-
-    with self.argument_context('iot pnp capability-model list') as context:
-        context.argument('search_string', options_list=['--search', '--ss'],
-                         help='Searches IoT Plug and Play models for given string in the'
-                              ' \"Description, DisplayName, comment and Id\".')
-        context.argument('top', type=int, options_list=['--top'],
-                         help='Maximum number of capability-model to return.')
+    with self.argument_context("iotcentral app monitor-events") as context:
+        context.argument(
+            "device_id", options_list=["--device-id", "-d"], help="Target Device."
+        )
+        context.argument("app_id", options_list=["--app-id"], help="Target App.")
+        context.argument(
+            "timeout",
+            options_list=["--timeout", "--to", "-t"],
+            type=int,
+            help="Maximum seconds to maintain connection without receiving message. Use 0 for infinity. ",
+        )
+        context.argument(
+            "consumer_group",
+            options_list=["--consumer-group", "--cg", "-c"],
+            help="Specify the consumer group to use when connecting to event hub endpoint.",
+        )
+        context.argument(
+            "enqueued_time",
+            options_list=["--enqueued-time", "--et", "-e"],
+            type=int,
+            help="Indicates the time that should be used as a starting point to read messages from the partitions. "
+            "Units are milliseconds since unix epoch. "
+            'If no time is indicated "now" is used.',
+        )
+        context.argument("properties", arg_type=event_msg_prop_type)
+        context.argument(
+            "content_type",
+            options_list=["--content-type", "--ct"],
+            help="Specify the Content-Type of the message payload to automatically format the output to that type.",
+        )
+        context.argument(
+            "repair",
+            options_list=["--repair", "-r"],
+            arg_type=get_three_state_flag(),
+            help="Reinstall uamqp dependency compatible with extension version. Default: false",
+        )
+        context.argument(
+            "central_api_uri",
+            options_list=["--central-api-uri"],
+            help="IoT Central API override.  For use with environments other than production.",
+        )
+        context.argument(
+            "yes",
+            options_list=["--yes", "-y"],
+            arg_type=get_three_state_flag(),
+            help="Skip user prompts. Indicates acceptance of dependency installation (if required). "
+            "Used primarily for automation scenarios. Default: false",
+        )
+
+    with self.argument_context("iotcentral device-twin show") as context:
```

Comment:
When is `iotcentral` going away? It is unfortunate to still see deprecated remnants.

Reply:
Chatted offline